src/builder/PaycallWrapper.sol

```diff
@@ -25,6 +25,7 @@ library PaycallWrapper {
 
         return IQuarkWallet.QuarkOperation({
             nonce: operation.nonce,
+            isReplayable: operation.isReplayable,
```

Comment:
This is interesting, and I think correct. The Paycall operation is repalyable IFF what it wraps is replayable?

Reply:
Exactly. If it's a `RecurringSwap` using `Paycall`, then the `Paycall` itself has to be replayable.